website/docs/table-group/data-table/data-table.md

```diff
@@ -89,14 +123,16 @@ If a cell is colored, it remains colored when you hover over it. Users shouldn't
 
 ## Cell content alignment
 
-Text inside cells in rows and headers is aligned according to these rules.
+**Vertical alignment:** both table header and body cell content is always aligned to the top.
+
+**Horizontall alignment:** cell content both in table header and body is aligned according to the following rules.
```

Comment:
Since we cannot control on the component level how all types of content in cells are aligned, maybe we should write it as a recommendation?

> **Horizontall alignment:** cell content both in table header and body should be aligned according to the following rules.

or

> **Horizontall alignment:** we recommend aligning cell content both in table header and body according to the following rules.

or

> **Horizontall alignment:** align cell content both in table header and body according to the following rules.